mtags/src/main/scala-3/scala/meta/internal/pc/CompilerSearchVisitor.scala

```diff
@@ -21,7 +22,12 @@ class CompilerSearchVisitor(
   val logger: Logger = Logger.getLogger(classOf[CompilerSearchVisitor].getName)
 
   private def isAccessible(sym: Symbol): Boolean = try
-    sym != NoSymbol && sym.isPublic && sym.isStatic
+    sym != NoSymbol && sym.isPublic && sym.isStatic || {
+      val owner = sym.maybeOwner
```

Comment:
can we extract this part to seperate method like `isAccessibleImplicit` ?

Reply:
We can also check whether the symbol is accessible from site, by using the compiler's `sym.isAccessibleFrom(qualType)` at CompilerSearchVisitor creation site

Reply:
Good idea @rochala though at this point it would be a much bigger refactor, so I will just move it to a method.